src/unix/bsd/netbsdlike/openbsd/mod.rs

```diff
@@ -383,6 +385,13 @@ s! {
         pub dlpi_phdr: *const Elf_Phdr,
         pub dlpi_phnum: Elf_Half,
     }
+
+    // pthread.h
+
+    pub struct __c_anonymous_pthread_spinlock {
```

Comment:
`pthread_spinlock` struct is internal and isn't in public header on purpose. the visible part is only a pointer, nothing else.

it is the same kind of type as `pthread_t` which is defined as `pub type pthread_t = ::uintptr_t` in Rust.